mvcc/kvstore.go

```diff
@@ -344,13 +327,14 @@ func (s *store) Restore(b backend.Backend) error {
 	close(s.stopc)
 	s.fifoSched.Stop()
 
-	atomic.StoreUint64(&s.consistentIndex, 0)
 	s.b = b
 	s.kvindex = newTreeIndex(s.lg)
 	s.currentRev = 1
 	s.compactMainRev = -1
 	s.fifoSched = schedule.NewFIFOScheduler()
 	s.stopc = make(chan struct{})
+	s.ci.SetBatchTx(b.BatchTx())
```

Comment:
Couple questions:
1. I believe we also need this for auth store (both newstore and recover)?
2. backend batchTx is recreated periodically (every 100ms or so), so we should pass backend instead of backend batchTx. Could you double check?

Reply:
1. auth store and mvcc store share the same consistent indexer, so it is not necessary for auth store.
2. batchTx.tx is recreated periodically, but we are passing batchTx to consistentIndexer, not batchTx.tx. There is only one function to reset batchTx (newBackend), so it looks ok.
![image](https://user-images.githubusercontent.com/2403673/77180865-85adf480-6b05-11ea-9fbe-9c1bda35f0e2.png)

Reply:
Sounds good. Thanks!